src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -165,31 +165,31 @@ public static void AssertPythonVersion(WorkerLanguageVersionInfo pythonVersion,
         {
             if (pythonVersion?.Version == null)
             {
-                var message = "Could not find a Python version. Python 3.6.x, 3.7.x, 3.8.x, 3.9.x, or 3.10.x is recommended, and used in Azure Functions.";
+                var message = "Could not find a Python version. Python 3.6.x, 3.7.x, 3.8.x, 3.9.x, 3.10.x or 3.11.x is recommended, and used in Azure Functions.";
```

Comment:
Lets remove Python 3.6 from the list. Its a v3 specific thing

Reply:
removed. thanks